addons/sourcemod/scripting/ttt/ttt_shop.sp

```diff
@@ -1431,6 +1543,19 @@ void ResetItemsArray(const char[] sFunction, bool initArray = false)
     }
 }
 
+bool GetItemLong(const char[] itemshort, char[] buffer, int size)
+{
+	any eItem[Item];
+	for (int i = 0; i < g_aCustomItems.Length; i++)
+	{
+		g_aCustomItems.GetArray(i, eItem[0]);
+		strcopy(buffer, size, itemshort);
```

Comment:
Seems like a check is missing wether the itemshort is missing the provided one + the function should return the itemlong not the itemshort.

Reply:
Oh God. Can't believe I forgot it.